skimage/filters/_gaussian.py

```diff
@@ -9,7 +11,7 @@
 __all__ = ['gaussian']
 
 
-def gaussian(image, sigma=1, output=None, mode='nearest', cval=0,
+def gaussian(image, sigma=1, output=None, mode='reflect', cval=0,
```

Comment:
Although this is a very small API change, please document it in ``release_dev.rst``.